crates/bootstrap_srv/src/bin/kitsune2-bootstrap-srv.rs

```diff
@@ -61,6 +61,25 @@ pub struct Args {
     /// Defaults: testing = 10s, production = 60s
     #[arg(long)]
     pub prune_interval_ms: Option<u32>,
+
+    /// Use this http header to determine IP address instead of the raw
+    /// TCP connection details.
+    #[cfg(feature = "sbd")]
+    #[arg(long)]
+    pub sbd_trusted_ip_header: Option<String>,
+
+    /// Limit client connections.
+    #[cfg(feature = "sbd")]
+    #[arg(long)]
+    pub sbd_limit_clients: Option<i32>,
+
+    /// If set, rate-limiting will be disabled on the server,
+    /// and clients will be informed they have an 8gbps rate limit.
+    ///
+    /// Note that this is an SBD option, but when SBD is enabled, this applies to all connections.
+    #[cfg(feature = "sbd")]
+    #[arg(long)]
+    pub sbd_disable_rate_limiting: bool,
```

Comment:
I'm totally cool with how this is currently set up.

But just musing: we will eventually need to support rate-limiting for bootstrapping anyways. It doesn't feel bad to me to always compile sbd in as a non-optional dependency to use its rate-limiting code, then optionally ignore websockets if something like a `--no-sbd` cli flag is passed.

Reply:
Yes great. I partly put the feature flag in as a caution thing, so that if I messed something up, we had an easy way to get rid of the code :D But if we want some of the functionality anyway and we're happy with this approach then I'm happy to make that change.

I'll jump into that on Monday :)